zkevm-circuits/src/evm_circuit/execution/codecopy.rs

```diff
@@ -203,6 +208,9 @@ impl<F: Field> ExecutionGadget<F> for CodeCopyGadget<F> {
             ),
         )?;
 
+        self.code_len_gadget
+            .assign(region, offset, block, &call.code_hash)?;
+
```

Comment:
This is unnecessary as the `code_len_gadget` has already been assigned witness to in L176

Reply:
already removed